src/components/LabelSlot.vue

```diff
@@ -427,6 +429,20 @@ export default Vue.extend({
             }
         },
         
+        handleMouseEnterLeave(isEntering: boolean) {
+            // There is a bug with how Firefox handles editable text HTML elements contained in a draggable div.
+            // We need to detect when the mouse is entering/leaving the text element to disable/enable the div's
+            // draggable attribute. 
+            // Because the frames are imbricated, we need to do that for all the frames hierarchy up ot the frames container.
```

Comment:
Today I learned a new word!  "Imbricated"...

Reply:
Aahahh yes? But it's good you mention this comment, because I think it's not correct - I really wanted to say "nested" so I'll change the word in the next commit.